chart/templates/_helpers.yaml

```
@@ -633,7 +633,13 @@ Create the name of the cleanup service account to use
 
 {{define "scheduler_liveness_check_command"}}
 
-  {{- if semverCompare ">=2.1.0" .Values.airflowVersion }}
+  {{- if semverCompare ">=2.4.0" .Values.airflowVersion }}
```

```suggestion
  {{- if semverCompare ">=2.5.0" .Values.airflowVersion }}
```

Sorry, we missed 2.4.0, and this will ship in 2.5.0 instead.